spec/lib/barcelona/plugins/ossec_client_plugin_spec.rb

```diff
@@ -0,0 +1,54 @@
+require "rails_helper"
```

Comment:
how about adding a spec to install all the plugins we use? Now that we have a bunch of plugins installed with some attributes, I'd like to know if there is no problem with the plugin combination we use.

Reply:
I think testing combination of plugins is a duty of integration tests. Bad configuration file or bad combination of packages can not be tested by unit tests anyway.

I have a plan to write a script or tool which tests a LaunchConfiguration. It will create a minimum ASG with a given LaunchConfiguration and an instance type. Then it verifies the EC2 instance launched on it.

And I think we can write it easily with AWS Step Function.

Reply:
sounds good. thanks